### ISSUE TITLE
Error when opening an empty css file

### ISSUE THREAD

**Issue by ismay** [AtomLinter/linter-stylelint#23]
```
error: You must pass stylelint a files glob or a code string, though not both
Error: You must pass stylelint a `files` glob or a `code` string, though not both
at Function.exports.default (/Users/ismay/.atom/packages/linter-stylelint/node_modules/stylelint/dist/standalone.js:50:11)
at /Users/ismay/.atom/packages/linter-stylelint/index.js:82:8
at Object.lint (/Users/ismay/.atom/packages/linter-stylelint/index.js:80:11)
at promises.push.Promise.then._this.emitter.emit.linter (/Users/ismay/.atom/packages/linter/lib/linter-registry.js:63:26)
at /Users/ismay/.atom/packages/linter/lib/linter-registry.js:62:23
at Set.forEach (native)
at LinterRegistry.lint (/Users/ismay/.atom/packages/linter/lib/linter-registry.js:60:18)
at /Users/ismay/.atom/packages/linter/lib/linter.coffee:14:3
at Emitter.module.exports.Emitter.emit (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:86:11)
at /Users/ismay/.atom/packages/linter/lib/editor-linter.js:80:22
at /Users/ismay/.atom/packages/linter/lib/helpers.coffee:23:18
```

Happened when I opened an empty .css file, without a stylelint present in the project. Stylelint is @1.2.2 and I've not selected `use preset`.

Let me know if you need more information!

**Comment by 1000ch** [comment]
Fixed in `v1.3.0`. Thank you for reporting!

**Comment by ismay** [comment]
:+1:
